src/features/metadata-instance-editor/__tests__/MetadataInstanceEditor.test.js

```
@@ -4,6 +4,7 @@ import { render, screen } from '../../../test-utils/testing-library';
 
 import MetadataInstanceEditor from '../MetadataInstanceEditor';
 import Instances from '../Instances';
+import { CASCADE_POLICY_TYPE_AI_EXTRACT } from '../constants';
```

probably better to use the literal values in the tests instead of reusing production constants

---

Is the though here that we want it to be easier to break?

---

yeah, if the constants change, we probably want to know about it in the test. sometimes it can silently pass for the wrong reasons